tests/storage/iso/test_iso_sr_reboot.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from lib.common import wait_for
+from conftest import copy_tools_iso_to_iso_sr, iso_mount_and_read_test_from_vm, clean_iso_sr, create_local_iso_sr
+
+# Requirements:
+# - one XCP-ng host with an additional unused disk for the SR
```

Comment:
And a VM.

Reply:
ok for th "And a VM.".

Could you explain your comment on the "tests/storage/iso/conftest.py" file ? I don't understand.